ansible/roles/pg_standby/templates/create_postgres_dump.sh.j2

```diff
@@ -3,7 +3,7 @@ BACKUP_TYPE=$1
 DAYS_TO_RETAIN_BACKUPS=$2
 
 TODAY=$(date +"%Y_%m_%d")
-pg_dumpall --clean | gzip  > "{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}.db.gz"
+pg_basebackup -D "{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}" -Ft -Xs -z
```

Comment:
I switched to filesystem level backups in our backup scripts because they are often faster to execute, but definitely faster to restore from. If both the main and standby servers go down restoring to the most recent backup is as simple as extracting the tar file and then turning on postgres.